### PR TITLE
Updated licensing docs with details of configuring an Umbraco application URL on Umbraco Cloud.

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Updated licensing docs with details of configuring an Umbraco application URL on Umbraco Cloud.  This is to clarify the situation and present options for the issue described in https://github.com/umbraco/Umbraco.Commerce.Issues/issues/440

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [X] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Workflow and Commerce

## Deadline (if relevant)

Can go live at any time.  Please can you wait for an approval from one or both of @mattbrailsford or @nathanwoulfe though.

### CODE REVIEW

File: 10/umbraco-commerce/getting-started/licensing-model.md
Comment (alina-tincas) [inline 10/umbraco-commerce/getting-started/licensing-model.md:101]:
```suggestion
If you are hosting on Umbraco Cloud you will find the configuration described above won't be reflected in your environment. The reason for this is that Umbraco Cloud sets this value as an environment variable set to the Cloud project domain (`<your project>.umbraco.io`). This overrides what is set via the `appSettings.json` file.

```

File: 10/umbraco-workflow/licensing.md
Comment (alina-tincas) [inline 10/umbraco-workflow/licensing.md:55]:
```suggestion
If you are hosting on Umbraco Cloud you will find the configuration described above won't be reflected in your environment. The reason for this is that Umbraco Cloud sets this value as an environment variable set to the Cloud project domain (`<your project>.umbraco.io`). This overrides what is set via the `appSettings.json` file.
```

File: 10/umbraco-commerce/getting-started/licensing-model.md
Comment (alina-tincas) [inline 10/umbraco-commerce/getting-started/licensing-model.md:107]: to make it more clear which options they are:
```suggestion
There are two options in this case:
- Either the domains for each of your Cloud environments can be added to your license.
- Or, for more control and to ensure this value is set correctly for other reasons, you can apply the configuration via code.
```

File: 10/umbraco-workflow/licensing.md
Comment (alina-tincas) [inline 10/umbraco-workflow/licensing.md:61]: to make it more clear which options they are:
```suggestion
There are two options in this case:
- Either the domains for each of your Cloud environments can be added to your license.
- Or, for more control and to ensure this value is set correctly for other reasons, you can apply the configuration via code.
```

File: 11/umbraco-workflow/licensing.md
Comment (alina-tincas) [inline 11/umbraco-workflow/licensing.md:55]:
```suggestion
If you are hosting on Umbraco Cloud you will find the configuration described above won't be reflected in your environment. The reason for this is that Umbraco Cloud sets this value as an environment variable set to the Cloud project domain (`<your project>.umbraco.io`). This overrides what is set via the `appSettings.json` file.
```

File: 11/umbraco-workflow/licensing.md
Comment (alina-tincas) [inline 11/umbraco-workflow/licensing.md:61]: to make it more clear which options they are:
```suggestion
There are two options in this case:
- Either the domains for each of your Cloud environments can be added to your license.
- Or, for more control and to ensure this value is set correctly for other reasons, you can apply the configuration via code.
```

File: 12/umbraco-workflow/licensing.md
Comment (alina-tincas) [inline 12/umbraco-workflow/licensing.md:55]:
```suggestion
If you are hosting on Umbraco Cloud you will find the configuration described above won't be reflected in your environment. The reason for this is that Umbraco Cloud sets this value as an environment variable set to the Cloud project domain (`<your project>.umbraco.io`). This overrides what is set via the `appSettings.json` file.
```

File: 12/umbraco-workflow/licensing.md
Comment (alina-tincas) [inline 12/umbraco-workflow/licensing.md:61]: to make it more clear which options they are:
```suggestion
There are two options in this case:
- Either the domains for each of your Cloud environments can be added to your license.
- Or, for more control and to ensure this value is set correctly for other reasons, you can apply the configuration via code.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @AndyButland thank you for the PR 🙌 

I have few suggestions can you look at them and let me know what you think? Also, I was wondering if perhaps v12 of commerce needs to have this updated as well? 

I will also await approval from either @mattbrailsford or @nathanwoulfe before merging this in 💪

**Comment by mattbrailsford** [comment]
Looks good to me 👍

**Comment by AndyButland** [comment]
Thanks @alina-tincas - I've accepted the suggestions and updated the Commerce 12 page now too.

**Pull request merged**
